.obs/dockerfile/slem4r-base-os/Dockerfile

```diff
@@ -69,18 +69,18 @@ RUN echo IMAGE_REPO=\"${IMAGE_REPO}\"              >> /etc/os-release && \
 # labelprefix=com.suse.sle.micro.rancher
 LABEL org.opencontainers.image.title="SLE Micro for Rancher"
 LABEL org.opencontainers.image.description="Image containing SLE Micro for Rancher - a containerized OS layer for Kubernetes."
-LABEL org.opencontainers.image.version="base-%%SLEMICRO_VERSION%%.%RELEASE%"
+LABEL org.opencontainers.image.version="${IMAGE_TAG}"
```

Comment:
As above I'd keep it equal to the most specific tag, so we can easily relate it with an specific build and source code version.

I am open to revisit the tag naming. The flavor explicit reference I understand it is already part of the image reference.

Also, and this is just a preference, I like using `-` as a separator of the different parts as this plays well for tags, makes fields easy to distinguish and matches the RPM scheme of `<version>-<release>`. The down side is it is not fully semver compliant...